integration_test/phantom/all_test.exs

```diff
@@ -0,0 +1,34 @@
+Code.require_file "../cases/browser_test.exs", __DIR__
+Code.require_file "../cases/browser/all_test.exs", __DIR__
+Code.require_file "../cases/browser/assert_css_test.exs", __DIR__
+Code.require_file "../cases/browser/assert_refute_has_test.exs", __DIR__
+Code.require_file "../cases/browser/assert_text_test.exs", __DIR__
+Code.require_file "../cases/browser/attr_test.exs", __DIR__
+Code.require_file "../cases/browser/check_test.exs", __DIR__
+Code.require_file "../cases/browser/clear_test.exs", __DIR__
+Code.require_file "../cases/browser/click_button_test.exs", __DIR__
+Code.require_file "../cases/browser/click_link_test.exs", __DIR__
+Code.require_file "../cases/browser/click_test.exs", __DIR__
+Code.require_file "../cases/browser/cookies_test.exs", __DIR__
+Code.require_file "../cases/browser/current_path_test.exs", __DIR__
+Code.require_file "../cases/browser/dialog_test.exs", __DIR__
+Code.require_file "../cases/browser/execute_script_test.exs", __DIR__
+Code.require_file "../cases/browser/file_test.exs", __DIR__
+Code.require_file "../cases/browser/fill_in_test.exs", __DIR__
+Code.require_file "../cases/browser/find_test.exs", __DIR__
+Code.require_file "../cases/browser/has_css_test.exs", __DIR__
+Code.require_file "../cases/browser/has_text_test.exs", __DIR__
+Code.require_file "../cases/browser/has_value_test.exs", __DIR__
+Code.require_file "../cases/browser/local_storage_test.exs", __DIR__
+Code.require_file "../cases/browser/navigation_test.exs", __DIR__
+Code.require_file "../cases/browser/page_source_test.exs", __DIR__
+Code.require_file "../cases/browser/screenshot_test.exs", __DIR__
+Code.require_file "../cases/browser/select_test.exs", __DIR__
+Code.require_file "../cases/browser/send_text_test.exs", __DIR__
+Code.require_file "../cases/browser/stale_nodes_test.exs", __DIR__
+Code.require_file "../cases/browser/text_test.exs", __DIR__
+Code.require_file "../cases/browser/title_test.exs", __DIR__
+Code.require_file "../cases/browser/visible_test.exs", __DIR__
+Code.require_file "../cases/browser/window_size_test.exs", __DIR__
+Code.require_file "../cases/element/send_keys_test.exs", __DIR__
+Code.require_file "../cases/query_test.exs", __DIR__
```

Comment:
I can see that scenario too, but this does allow us to skip a test case that's not supported by a specific driver. Maybe another option would be to use a tag based approach (like you do with the pending examples), but I'm trying to figure out what that would look like.

```elixir
test "phantom specific behavior", only_drivers: ["phantom"] do

end
```
or 
```elixir
test "behavior that isn't supported by phantom", except_drivers: ["chrome"] do

end
```

At first glance, I'm not that big a fan of this approach either. Especially if we accidentaly had`only_drivers` and `except_drivers` on the same test case.

Any ideas on how to implement this and still have control on which tests run against which drivers would be appreciated. 🙂